tlslite/utils/codec.py

```diff
@@ -45,17 +209,24 @@ def addVarTupleSeq(self, seq, length, lengthLength):
         @type lengthLength: int
         @param lengthLength: length in bytes of overall length field
         """
-        if len(seq) == 0:
+        if not seq:
```

Comment:
Is this supposed to be None, or an empty enum? I detest the ambiguity of just using "not" in python as it can mean many many things.